root_draft_metadata.tf

```diff
@@ -46,9 +47,212 @@ module "draft_metadata_api_gateway" {
   }]
 }
 
+resource "aws_iam_role" "draft_metadata_api_gateway_execution_role" {
+  name = "TDRMetadataChecksAPIGatewayExecutionRole${title(local.environment)}"
+
+  assume_role_policy = jsonencode({
```

Comment:
TDR uses json files instead of directly having policies defined in the `.tf` files themselves.